src/storekit.cs

```diff
@@ -774,7 +778,7 @@ interface SKCloudServiceController {
 		[Static]
 		[Async]
 		[Export ("requestAuthorization:")]
-		void RequestAuthorization (Action<SKCloudServiceAuthorizationStatus> handler);
+		void RequestAuthorization (Action<SKCloudServiceAuthorizationStatus> completionHandler);
```

Comment:
it's better but it's a breaking change - it's not the worse kind but it's better avoided unless something else force us to make that change